.github/workflows/swift-toolchain.yml

```diff
@@ -2129,16 +2239,28 @@ jobs:
         arch: ['amd64' , 'arm64']
 
     steps:
+      - name: Download Debugging Tools
+        uses: actions/download-artifact@v4
+        # TODO(thebrowsercompany/swift-build#125): Fix the arm64 build and uncomment this.
+        if: matrix.arch == 'amd64'
+        with:
+          name: debugging_tools-${{ matrix.arch }}
+          path: ${{ github.workspace }}/BuildRoot/Library
+
       - name: Download Compilers
         uses: actions/download-artifact@v4
         with:
           name: compilers-${{ matrix.arch }}
           path: ${{ github.workspace }}/BuildRoot/Library
-      - uses: actions/download-artifact@v4
+
+      - name: Download Devtools
```

Comment:
I think that the stage might be `DevTools`.

Reply:
Fixed capitalization